chart/k8gb/values.yaml

```diff
@@ -92,3 +92,6 @@ route53:
   enabled: false
   hostedZoneID: ZXXXSSS
   irsaRole: arn:aws:iam::111111:role/external-dns
+
+ns1:
+  enabled: true
```

Comment:
Should this value be set to `false` by default?
Also, my understanding is that we support one EdgeDNS provider at runtime.
It therefore makes sense to introduce general `edgeDNSType` parameter in confifuration and update logic in  depresolver's `getEdgeDNSType()` accordingly, wdyt?

Reply:
it should be, that's why it is labeled WIP :)

Reply:
not sure about `edgeDNSType`. Options heavily deviate from provider to provider and frequently not compatible

Reply:
You're right, it can be `provider` or `providerType`. 
My understanding is that options `infoblox.enabled`, `route53.enabled`,  `ns1.enabled` are mutually exclusive, and can be replaced with that one option in order to avoid confusion in the future, when more providers are added.

Reply:
Hi, agree with @somaritane. For now, depresolver acts as [facade](https://en.wikipedia.org/wiki/Facade_pattern), so transforms to `edgeDNSType` internally. To make it cleaner, we should extract parameter directly into configuration.

Reply:
@somaritane @kuritka it means mixing up totally unrelated configuration options under the same hierarchy level. I would be very careful with that. We can attack it in a separate issue

Reply:
The Issue for the above https://github.com/AbsaOSS/k8gb/issues/219